collector/network/slurm/slurm_custom_ar_16gpu.sh

```
@@ -0,0 +1,33 @@
+# Cross-node MNNVL allreduce (same recipe as the NCCL multi-node sweep)
+export NCCL_NVLS_ENABLE=1
+export NCCL_MNNVL_ENABLE=1
+export NCCL_CUMEM_ENABLE=1
+export NCCL_CUMEM_HOST_ENABLE=0
+export TRTLLM_DEEPSEEK_EAGER_FUSION_DISABLED=0
+
+# collect_allreduce.py reads SLURM_NTASKS / SLURM_NTASKS_PER_NODE / SLURM_LOCALID / RANK.
+srun -l \
+    --ntasks 16 --ntasks-per-node 4 \
+    --container-image=/path/to/trtllm_aarch64_release_v1.0.0rc2.sqsh \
+    --container-mounts=/dev:/dev,${HOME}:${HOME},/path/to/:/kimi \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)